### DISCUSSION TITLE
Plotting Different Output Variables

### DISCUSSION THREAD

**Discussion by adamleik** [MoiseRousseau/PyGeoStudio#5] [Ideas]
Hi, I am hoping to use your library to plot simulations of element tests using Sigma/w. I am wanting to plot variables other than 

['Node',
 'XDisplacement',
 'YDisplacement',
 'PoreWaterPressure',
 'XDisplacementIncrement',
 'YDisplacementIncrement']

I can't seem to determine why the existing code only pulls in these variables as options. Have you attempted to plot things like deviatoric stress or mean effective stress?

**Comment by MoiseRousseau** [comment]
Hi,

Currently, the plotting variables are read from the results of GeoStudio simulation at the mesh nodes. It seems however that the stresses are stored at gauss integration points, for which I do not have the coordinates, or the coordinates is not written in GeoStudio output file. I should figure out more where they are located to be able to plot them.

Moise
